src/content/docs/build/env-variables/add-manage-env-variables.mdx

```diff
@@ -0,0 +1,40 @@
+---
+page_id: 82646df5-5e4c-4b09-babf-2f77c36bd73b
+title: Add and update environment variables
+sidebar:
+  order: 2
+relatedArticles:
+  - 2d39a390-baf8-4f49-a18c-950d3f248c65
+  - d95a8edd-f61a-4095-a80e-ffd67f90a438
+---
+
+Environment variables are for including sensitive information in your application code, using a key. You then update the variable's key-value to update the value wherever the key is referenced. You can store as many environment variables in Kinde as you want. 
+
+## Recommendations for variables
+
+- Choose a key name that helps you easily recognize what the variable is for.
+- Use a consistent case, such as CamelCase, snake_case, kebab-case, etc.
+- Sensitive key values are obscured for security, but they can still be copied.
```

Comment:
@clairekinde11 Sensitive variables cannot be copied. 

Sensitive environment variables values are non-readable once created. They help protect sensitive information stored in environment variables, such as API keys.

Reply:
I would say the recommendation is
- Mark a variable as sensitive if the value should be kept secret. For example an API key or password.

Reply:
Updated.